### PR TITLE
Version 0.3 beta

### PR DESCRIPTION
**Pull request by tomchristie**
- [x] change Schema to plain instances (by Tom Christie)
- [x]  minor modifications, add tests and uplift coverage
- [x] update documentation
- [x] update examples

### CODE REVIEW

**Comment by tomchristie** [comment]
@aminalaee When you're ready to roll a release, you'll want to create a release draft that we can check over...

Screenshot 2021-08-18 at 14 51 04

Screenshot 2021-08-18 at 14 51 20

Once we're happy with that, we can:

* Merge the new version pull request...
* ...then publish the release.

**Comment by aminalaee** [comment]
@tomchristie Thank you. I'm a little confused now.
Do we want to merge this beta version into master now? Shouldn't we publish beta releases `0.3.0.devX` of this branch instead of master, and when it's ready merge into master and close this PR and publish `0.3.0`?

**Comment by tomchristie** [comment]
@aminalaee - So I wasn't sure of the answer to that question. The sensible thing here is "do whatever we're currently already doing". I'd assumed `master` branch was on 0.3.0.dev0, but [I can see now that's not the case](https://github.com/encode/typesystem/blob/master/typesystem/__init__.py), and that 0.3 pre-releases are [maintained on a separate branch](https://github.com/encode/typesystem/blob/version-0.3-beta/typesystem/__init__.py).

> Shouldn't we keep this branch, publish beta releases 0.3.0.devX of this instead of master, and when it's ready, merge into master and close this PR and publish 0.3.0?

Yes that would make sense.

File: docs/forms.md [hunk 1]
```diff
@@ -34,39 +38,39 @@ Notice that only the fields in the form are rendered. The surrounding `<form>`,
...
     </td>
 </tr>
 <tr>
     <td>
-        <label for="form-bookingschema-end-date">End date</label>
+        <label for="">End date</label>
     </td>
     <td>
-        <input type="date" id="form-bookingschema-end-date" name="end_date" required>
+        <input type="date" id="" name="end_date" required >
```
Comment (aminalaee) [inline docs/forms.md:52]: `id` and `label` fields are empty.

**Review by aminalaee — CHANGES_REQUESTED** [review]

File: docs/forms.md (hunk 1 above)
Comment (aminalaee) [reply docs/forms.md]: Added `label` and `id` without prefix

**Comment by aminalaee** [comment]
https://github.com/encode/typesystem/releases/edit/untagged-a8553b8dccc475111fef

**Review by aminalaee — APPROVED** [review]

**Pull request merged**